front/src/components/article/articlesFilterFavoritesRow.component.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+
+import { Labels } from "../../constants";
+import { Colors, Margins, Paddings, Sizes } from "../../styles";
+import { SecondaryText, Toggle } from "../baseComponents";
+
+interface Props {
+  isToggleOn: boolean;
+  onToggleFavorite: () => void;
+}
+
+const ArticlesFilterFavoritesRow: React.FC<Props> = ({
+  isToggleOn,
+  onToggleFavorite,
+}) => {
+  return (
+    <View style={styles.mainContainer}>
+      <SecondaryText
+        style={{
+          color: Colors.primaryBlueDark,
+          fontSize: Sizes.xs,
+          fontWeight: isToggleOn ? "bold" : "normal",
```

Comment:
```suggestion
          fontWeight: isToggleOn ? FontWeight.bold : FontWeight.normal,
```
Il y a l'import `FontWeight` à rajouter je crois